.coveragerc

```diff
@@ -0,0 +1,13 @@
+[run]
+branch = True
+source = coramin
+
+[report]
+exclude_lines =
+    if self.debug:
+    pragma: no cover
+    raise NotImplementedError
+    if __name__ == .__main__.:
+ignore_errors = True
+omit =
+    tests/*
```

Comment:
I think you need a newline at the end of this file.

Reply:
Good catch. Thanks.